### PR TITLE
Add library image for notary-signer

### PR DESCRIPTION
**Pull request by aaronlehmann**
This is based on swarm-library-image.

Unfortunately, the notary-server binary needs to be checked in -
otherwise "docker build" wouldn't work without preparation, as is
required of official images. The binary checked in here is from notary
a95349457ab2a1f36209596bebe9464dc719a335. It will need to be updated to
the release version after each release (using the update.sh script).

Signed-off-by: Aaron Lehmann aaron.lehmann@docker.com

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Build a notary-signer container without the build environment
+
+FROM ubuntu:14.04
+
+RUN apt-get update && \
+    apt-get install -y libltdl7 sqlite3 ca-certificates && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN buildDeps=' \
```
Comment (jessfraz) [inline Dockerfile:9]: this looks familiar i wonder who wrote this damn fine looking function :angel:

**Comment by jessfraz** [comment]
nice work!

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Build a notary-signer container without the build environment
+
+FROM ubuntu:14.04
```
Comment (jessfraz) [inline Dockerfile:3]: why not debian, its smaller

Comment (aaronlehmann) [inline Dockerfile:3]: Using debian seems reasonable. I can try that.

**Comment by aaronlehmann** [comment]
I guess this never got merged. Can't be very important, then.

**Pull request closed without merging**